src/noit_filters.c

```diff
@@ -328,6 +339,25 @@ noit_filter_compile_add(mtev_conf_section_t setinfo) {
       rule->type = NOIT_FILTER_SKIPTO;
       rule->skipto = strdup(buffer+strlen(FILTERSET_SKIPTO_STRING));
     }
+    else if (!strncasecmp(buffer, FILTERSET_ADD_MEASUREMENT_TAG_STRING, strlen(FILTERSET_ADD_MEASUREMENT_TAG_STRING))) {
```

Comment:
here you are not case sensitive, but in other places it uses strncmp, is this intentional?

Reply:
I'm matching the style of what's already here. If we change the style on the new code, we should probably change it on everything, which would theoretically be done in another PR